drivers/ieee802154/Kconfig.tlx

```diff
@@ -48,6 +48,13 @@ config IEEE802154_TLX_DELAY_TRX_ACC
 	  Accuracy of the clock used for scheduling radio delayed operations (delayed transmission
 	  or delayed reception), in ppm.
 
+config IEEE802154_TLX_OPTIMIZATION
```

Comment:
config IEEE802154_TLX_OPTIMIZATION
	bool "Optimize 802.15.4 RF performance for TLX SoCs"
	default n
	help
	  Improve RF performance in IEEE 802.15.4 communication on TLX System-on-Chips (SoCs).

Reply:
Updated.